app/views/observer/_form_observation_notes.html.erb

```diff
@@ -0,0 +1,70 @@
+<% # Notes section of create_observation form %>
+<div class="row push-down">
+  <%= content_tag(:span, "#{:NOTES.t} ", class: "bold") %>
+  <% if @observation.form_notes_parts(@user) == [Observation.other_notes_part]
+       part = Observation.other_notes_part %>
+
+    <div class="row">
+      <div class="col-xs-12 col-sm-6 max-width-text">
+        <div>
+          <textarea class="form-control" rows="10"
+          id=<%= @observation.notes_part_id(part) %>
+          name=<%= @observation.notes_part_name(part) %>
+          ><%= @observation.notes_part_value(part) %></textarea>
```

Comment:
I agree with @pellaea that the HTML should be in the template rather than moving to a custom helper.

But I wonder if I could avoid the HTML entirely by using a combination of:
- a Rails' form helper to create the textarea (resulting in areas which have ids and names **not** prefixed by `observation`, and 
- `wrap_parameters` to read the values in the controller as `params[:observation][:notes]` rather than `params[:notes]`.